tools/source-sync/source_sync.py

```diff
@@ -0,0 +1,192 @@
+"""Download all of the records from one or all of the defined OSV.dev sources"""
+
+import os
+import argparse
+import yaml
+import json
+import sys
+import pprint
+import urllib
+from urllib3.util.retry import Retry
+import requests
+from requests.adapters import HTTPAdapter
+import re
+import pygit2
+from google.cloud import storage
+from google.cloud.storage import retry
+
+
+class Error(Exception):
```

Comment:
nit: use a docstring here (and then we can also get rid of the `pass`)

Reply:
Done in https://github.com/google/osv.dev/pull/2416/commits/159714a2edc34dfdd1d0b92fb6f9cdddfce632ee